business/checkers/common/export_to_namespace_checker_test.go

```diff
@@ -0,0 +1,55 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/models"
+	"github.com/kiali/kiali/tests/data"
+)
+
+func TestNamespaceExist(t *testing.T) {
+	assert := assert.New(t)
+
+	validations, valid := ExportToNamespaceChecker{
+		IstioObject: createDestinationRule([]interface{}{"bookinfo", "bookinfo2", "."}),
+		Namespaces: models.Namespaces{
+			models.Namespace{Name: "bookinfo"},
+			models.Namespace{Name: "bookinfo2"},
+			models.Namespace{Name: "default"},
+		},
+	}.Check()
+
+	// Well configured object
+	assert.True(valid)
+	assert.Empty(validations)
+}
+
+func TestNamespaceNotFound(t *testing.T) {
+	assert := assert.New(t)
+
+	validations, valid := ExportToNamespaceChecker{
+		IstioObject: createDestinationRule([]interface{}{"wrong1", "bookinfo3", "."}),
+		Namespaces: models.Namespaces{
+			models.Namespace{Name: "bookinfo"},
+			models.Namespace{Name: "bookinfo2"},
+		},
+	}.Check()
+
+	assert.False(valid)
+	assert.NotEmpty(validations)
+	assert.Len(validations, 2)
+	assert.Equal(validations[0].Message, models.CheckMessage("generic.exportto.namespacenotfound"))
+	assert.Equal(validations[0].Severity, models.ErrorSeverity)
+	assert.Equal(validations[0].Path, "spec/exportTo[0]")
+	assert.Equal(validations[1].Message, models.CheckMessage("generic.exportto.namespacenotfound"))
+	assert.Equal(validations[1].Severity, models.ErrorSeverity)
+	assert.Equal(validations[1].Path, "spec/exportTo[1]")
+}
```

Comment:
If you are using the `YamlFixtureLoader` it'll be quite easy to validate not only destinationrules but ServiceEntries and VirtualServices. So I'd add one test case for each resource. To make sure everything is alright.

Reply:
+1